aws/rust-runtime/aws-config/src/imds/client.rs

```diff
@@ -984,7 +984,7 @@ pub(crate) mod test {
     #[cfg_attr(windows, ignore)]
     /// Verify that the end-to-end real client has a 1-second connect timeout
     #[tokio::test]
-    #[cfg(feature = "rustls")]
+    #[cfg(feature = "default-http-connector")]
```

Comment:
I believe it's ok to not include the `rustls` feature in tests unlike production code?

Reply:
Yeah I left it as `any(feature = "default-http-connector", feature = "rustls"))` on public code as I didn't want someone relying on `--no-default-features features=rustls` for example to break.